src/metabase/public_settings/metastore.clj

```diff
@@ -154,35 +154,35 @@
   :type       :boolean
   :visibility :public
   :setter     :none
-  :getter     (fn [] (boolean ((token-features) "embedding"))))
+  :getter     (fn [] (and config/ee-available? (boolean ((token-features) "embedding")))))
```

Comment:
I think this might be the one token feature that we do something with from OSS. Maybe grep the codebase and see if we use `enable-embedding?` or if the FE checks for the `embedding` token feature in any OSS code?